### DISCUSSION TITLE
Where is the repository for the home page https://nextjs.org/ ?

### DISCUSSION THREAD

**Discussion by entest-hai** [vercel/next.js#41350] [Help]
Please tell me where is the Git repository for the home page? I would like to look into it. I can find the docs part but can't find the home page index.*

**Comment by icyJoseph** [comment]
I think it is not open sourced. I once tried to figure out how the docs are build, because I wanted to add copy-to-clipboard on the code snippets, but could never find the actual site repository.

**Reply by entest-hai** [reply]
thank you!
